retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc

```diff
@@ -26,7 +26,7 @@ The following objects must be created manually or via Apex script:
 
 === Create Custom Object via Apex Script
 
-The`CustomObjectUtility` and `MetadataService` Apex classes are used to create a custom object programmatically. The source can be found on https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources/[RCG Common Resources] project and https://github.com/financialforcedev/apex-mdapi/blob/master/apex-mdapi/src/classes/MetadataService.cls[MetadataService] respectively. Download the project and follow the steps below.
+The`CustomObjectUtility` and `MetadataService` Apex classes are used to create a custom object programmatically. The source can be found on https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project and https://github.com/financialforcedev/apex-mdapi/blob/master/apex-mdapi/src/classes/MetadataService.cls[MetadataService] respectively. Download the project and follow the steps below.
```

Comment:
```suggestion
Use the`CustomObjectUtility` and `MetadataService` Apex classes to create a custom object programmatically. The source can be found on the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project and https://github.com/financialforcedev/apex-mdapi/blob/master/apex-mdapi/src/classes/MetadataService.cls[MetadataService] respectively. Download the project and follow these steps:
```

Reply:
This is one of those changes that needs to be back-ported to beta.